src/content/books/lost-approachable-open-source.md

```diff
@@ -0,0 +1,11 @@
+---
+collection: lost
+number: 0
+title: Approachable Open Source
+author: brian-muenzenmeyer
+url: https://approachableopensource.com
+color: "#000"
+isbn: 979-8-9907798-0-8
+pages: 190
+published: 2024-08-27
```

Comment:
What's the basis for this date, if I may ask? Since ABA never published these 'lost' books, I could go with their planned pub dates (but those may not have been decided), the dates books were shipped with other publishers (which covers at least one lost book), or the dates that contracts were signed with ABA. But my instinct was to keep it simple and have them in alphabetical order at the end of the list. WDYT?

Reply:
This is the official publish date entered in bowker.

I'm happy to have them included in any way, alphabetical makes sense to me.

Of the 3 known lost books that will see the light of day, we think I am the middle chronologically:

1. [Sustainable Content](https://bookshop.org/p/books/sustainable-content-how-to-measure-and-mitigate-the-carbon-footprint-of-digital-data-alisa-bonsignore/21976900?ean=9781632998828)
2. [Approachable Open Source](https://approachableopensource.com/)
3. [Designing Content Authoring Experiences](https://www.kickstarter.com/projects/gregdunlap/designing-content-authoring-experiences)